src/gluonnlp/model/transformer.py

```diff
@@ -1208,3 +1209,35 @@ def transformer_en_de_512(dataset_name=None, src_vocab=None, tgt_vocab=None, pre
                                   predefined_args['share_embed'], predefined_args['embed_size'],
                                   predefined_args['tie_weights'],
                                   predefined_args['embed_initializer'], pretrained, ctx, root)
+
+
+class ParallelTransformer(Parallelizable):
+    """ Data parallel transformer.
+
+    Parameters
+    ----------
+    model : Block
+        The transformer model.
+    label_smoothing: Block
+        The block to perform label smoothing.
+    loss_function : Block
+        The loss function to optimizer.
+    rescale_loss : float
+        The scale to which the loss is rescaled to avoid gradient explosion.
+    """
+    def __init__(self, model, label_smoothing, loss_function, rescale_loss):
+        self._model = model
+        self._label_smoothing = label_smoothing
+        self._loss = loss_function
+        self._rescale_loss = rescale_loss
+
+    def forward_backward(self, x):
```

Comment:
Do you know why pylint does not complain about missing docstring here?

Reply:
Maybe due to https://github.com/dmlc/gluon-nlp/blob/master/.pylintrc#L327-L329 ?